rotkehlchen/serialization/deserialize.py

```diff
@@ -325,6 +327,27 @@ def deserialize_asset_movement_category(
     )
 
 
+def deserialize_asset_movement_event_type(
+        value: str,
+) -> Literal[HistoryEventType.DEPOSIT, HistoryEventType.WITHDRAWAL]:
+    """Takes a string and determines whether to accept it as an asset movement event_type
+
+    Can throw DeserializationError if value is not as expected
+    """
+    if isinstance(value, str):
+        if value.lower() == 'deposit':
+            return HistoryEventType.DEPOSIT
+        if value.lower() in {'withdraw', 'withdrawal'}:
+            return HistoryEventType.WITHDRAWAL
+        raise DeserializationError(
+            f'Failed to deserialize asset movement category symbol. Unknown {value}',
+        )
```

Comment:
```suggestion
        if (lowered_value := value.lower()) == 'deposit':
            return HistoryEventType.DEPOSIT
        if lowered_value in {'withdraw', 'withdrawal'}:
            return HistoryEventType.WITHDRAWAL
        raise DeserializationError(
            f'Failed to deserialize asset movement category symbol. Unknown {value}',
        )
```

Reply:
We already have `deserialize_asset_movement_category`. I believe you took it from there right?

Reply:
Yes. `deserialize_asset_movement_category` will be removed after all the exchange interfaces are converted. I added a line to its doc string mentioning that it is deprecated.